eo-maven-plugin/src/main/java/org/eolang/maven/Unplace.java

```diff
@@ -62,7 +86,7 @@ final class Unplace {
     public String make(final Path file) {
         return file.toString().substring(
             this.parent.toString().length() + 1
-        ).replaceAll(".eo$", "").replace(File.separator, ".");
+        ).replaceAll(this.extension, "").replace(File.separator, ".");
```

Comment:
@ArtemGet this looks pretty dangerous. What if `this.extension` will be equal to `(` -- you will get runtime exception here. Try using `Pattern.quote()`.

Reply:
@yegor256 Agreed, add Pattern.quote and tests.